readthedocs/proxito/urls.py

```diff
@@ -50,6 +51,13 @@
 
 DOC_PATH_PREFIX = getattr(settings, 'DOC_PATH_PREFIX', '')
 
+health_check_urls = [
+    url('^{DOC_PATH_PREFIX}health_check/$'.format(DOC_PATH_PREFIX=DOC_PATH_PREFIX),
```

Comment:
It seems we have this health check for El Proxito process. Does it worth to have the same health check for Web process as well?

Reply:
@humitos The health checks currently are at the server level, so proxito is all we need since it's the default in the nginx config.